in lib3
Running f1 from lib3
Running f2 from lib3
finished with lib3
Running f1 from lib3
Running f2 from lib3
finished loading lib2
test loaded
in lib1
Running f1 from lib3
Running f1 from lib3
Running f2 from lib3
Running f3 from lib2
Running f4 from lib2
Running f4 from lib2
In main
Running f1 from lib3
Running f2 from lib3
Running f3 from lib2
Running f4 from main!
constant c1's value: 1
Running f1 from lib3
Running f2 from lib3
Running f1 from lib3
Running f2 from lib3
Running sublib fn1
Running sublib fn1
Running f1 from lib3
Running f2 from lib3
Running subdir/subdir2/sublib fn2
Running subdir/subdir2/sublib fn2
with an object now
Running sublib fn1
Running sublib fn1
Running sublib fn1000
Running sublib fn1
error: attempt to call non-procedure
	#0                    anon	(lib_library.l1:73)
	#1                   entry	(lib_library.l1:76)
expected error found
Testing lone object import
Running sublib fn1
in lib1
